exporter/elasticsearchexporter/model.go

```diff
@@ -107,3 +108,13 @@ func spanLinksToString(spanLinkSlice ptrace.SpanLinkSlice) string {
 func durationAsMicroseconds(start, end time.Time) int64 {
 	return (end.UnixNano() - start.UnixNano()) / 1000
 }
+
+func scopeToAttributes(scope pcommon.InstrumentationScope) pcommon.Map {
+	attrs := pcommon.NewMap()
+	attrs.PutStr("name", scope.Name())
```

Comment:
I'm not familiar with this component, so just checking to make sure it's alright: I see the attribute name here doesn't match with the filed issue's name (`name` vs. `otel.library.name`), is that acceptable here?  I assume this won't cause any problems since backends have their own expectations around naming schemas, but just wanted to confirm.

Reply:
`otel.library.name` was defined by jaeger itself(https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/fa8883772f61054fd626dbc4a06146dc6ec09999/pkg/translator/jaeger/traces_to_jaegerproto.go#L387), so, we could not follow it.